.travis.yml

```diff
@@ -77,7 +77,7 @@ jobs:
         - mv src/e2e/resources/test.travis.properties src/e2e/resources/test.properties
         - ./gradlew createConfigs testClasses generateTypes
         - npm ci
-        - npm run build -- --progress=false
+        - npm run build -- --progress=false --serviceWorker=false
```

Comment:
I believe this is by default false so I am not very sure if it is needed here?

Reply:
https://github.com/TEAMMATES/teammates/blob/4cce50f625ceb0e411d34168424bab7c546cca3a/angular.json#L106